mucommander-core/src/main/java/com/mucommander/ui/main/table/FileTableModel.java

```diff
@@ -93,18 +120,27 @@
     /**
      * Sets the SizeFormat format used to create the size column's string.
      *
-     * @param compactSize true to use a compact size format, false for full size in bytes 
+     * @param compactSize true to use a compact size format, false for full size in bytes
      */
     static void setSizeFormat(boolean compactSize) {
         if(compactSize)
             sizeFormat = SizeFormat.DIGITS_MEDIUM | SizeFormat.UNIT_SHORT | SizeFormat.ROUND_TO_KB;
         else
-            sizeFormat = SizeFormat.DIGITS_FULL | SizeFormat.UNIT_NONE;
+        sizeFormat = SizeFormat.DIGITS_FULL;
```

Comment:
the indentation doesn't look right here, no?
and why changing that?

Reply:
my bad copy/pasting, was porting trolCommander's commit late night, will fix it.